utils/src/treev2/quickPickWizard/RecursiveQuickPickStep.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as types from '../../../index';
+import { ContextValueFilterQuickPickOptions, ContextValueQuickPickStep } from './ContextValueQuickPickStep';
+import { getLastNode } from './QuickPickWizardContext';
+
+export class RecursiveQuickPickStep<TContext extends types.QuickPickWizardContext> extends ContextValueQuickPickStep<TContext, ContextValueFilterQuickPickOptions> {

```

Comment:
Even though you return a new step with `hideStepCount` set to `true`, we should just define it as its default property. Otherwise wouldn't the first step has a step count and then it just disappears?

Reply:
Makes sense. Although, all the wizards using this step have `hideStepCount` set to `true`.

Reply:
I didn't realize that `hideStepCount` could be a property of the steps themselves, else I'd have done it that way from the start. :smile: